docs/conf.py

```diff
@@ -124,7 +124,7 @@ def _skip(self, word):
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
Is this still necessary, even with the dependencies pinning?

Reply:
Indeed, not needed